test/DefaultEmissionManager.t.sol

```diff
@@ -224,6 +226,6 @@ contract DefaultEmissionManagerTest is Test {
         inputs[2] = vm.toString(delay);
         inputs[3] = vm.toString(polygon.totalSupply());
         uint256 newSupply = abi.decode(vm.ffi(inputs), (uint256));
-        assertApproxEqAbs(newSupply, emissionManager.inflatedSupplyAfter(block.timestamp + delay), 1e20);
```

Comment:
Some assertions have been removed without further explanation like removing += by = as well as block.timestamp dependencies which is not good as we are just testing within a block now (not recommended for critical production env)

Reply:
I added this comment in line 200:
```
// now that we actually pass this to calc.js, we only need to set it once.
        uint256 initialTotalSupply = polygon.totalSupply();
```

before, calc.js had `startSupply` hardcoded at `10_000_000_000e18`
now that our value actually gets parsed, we only set `initialTotalSupply` one time at the beginning.
This leads to us also not having to increment the `balance` and `stakeManagerBalance` anymore, but simply setting them to the new value, because when we calculate `totalAmtMinted`, `initialTotalSupply` now stays the same. Hope that makes sense :)